index.js

```diff
@@ -27,7 +27,8 @@ function unauthorized(res) {
     message = removePrefix(err.message, ':'),
     code = 401;
 
-  res.stats(code).json({ code, message });
+  res.statusCode = code;
+  res.json({ code, message });
```

Comment:
You can change this back
```suggestion
  res.status(code).json({ code, message });
```